packages/core/src/auth/auth.ts

```diff
@@ -502,8 +506,10 @@ export class Auth implements AuthService, ConnectionManager {
     private async validateConnection<T extends Profile>(id: Connection['id'], profile: StoredProfile<T>) {
         const runCheck = async () => {
             if (profile.type === 'sso') {
+                getLogger().info(`auth: Validating SSO connection: ${id}`)
                 const provider = this.getTokenProvider(id, profile)
                 if ((await provider.getToken()) === undefined) {
+                    getLogger().info(`auth: Connection is not valid: ${id} `)
                     return this.updateConnectionState(id, 'invalid')
                 } else {
                     return this.updateConnectionState(id, 'valid')
```

Comment:
Let's add a "Connect is valid" log here. And remove the "Validating..." log on line 509. That gives us something explicit to look for in both cases.

Reply:
DONE